src/test/java/com/notably/commons/core/Path/AbsolutePathTest.java

```diff
@@ -0,0 +1,82 @@
+package com.notably.commons.core.path;
+
+import static com.notably.testutil.Assert.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import com.notably.commons.core.path.exceptions.InvalidPathException;
+
+
+class AbsolutePathTest {
+
+    @Test
+    public void createAbsolutePath_validInputString_generateAbsolutePath() {
```

Comment:
Hmmm, two things:
1. You can actually don't care about the exception here. Just make the `createAbsolutePath_validInputString_generateAbsolutePath` method throws `InvalidPathException` since you know for sure that this method shouldn't throw if it functions correctly.
2. I don't think testing the correct creation of `AbsolutePath` this way actually tests anything significant. That is because we know both `expectedOutput` and `testInput` are created (either explicitly or implicitly) using `AbsolutePath`'s constructor! Instead, I propose we do the testing as such:

```java
final String pathString = "/CS2103/notes";

final List<String> expectedComponents = List.of("CS2103", "notes");

AbsolutePath createdPath = AbsolutePath.fromString(pathString);
List<String> createdComponents = createdPath.getComponents();
assertEquals(expectedComponents, createdComponents);
```